packages/backend/src/extCommands.ts

```diff
@@ -29,32 +29,64 @@ export class ExtCommands {
   }
 
   private async yeomanUIPanel_runGenerator_Command() {
-    return (await this.getYeomanUIPanel()).runGenerator();
+    try {
+      return (await this.getYeomanUIPanel()).runGenerator();
+    } catch (e) {
+      console.log(e);
```

Comment:
there is no meaning of console.log in the backend, there is already a logger you can use if you want